.storybook/Absolute.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import styled from 'styled-components'
+import { Absolute, Box, Card, Flag, Flex, Icon, Image, Text } from '../src'
+
+const RelativeBox = styled(Box)`
+  position: relative;
+`
+
+const RelativeCard = styled(Card)`
+  position: relative;
+`
+
+const TallCard = styled(RelativeCard)`
+  height: 420px;
+`
+
+const SmallAbsolute = styled(Absolute)`
+  height: 100px;
+`
+
+const MediumAbsolute = styled(Absolute)`
+  height: 200px;
+`
+
+const LargeAbsolute = styled(Absolute)`
+  height: 300px;
+`
+
+const ExtraLargeAbsolute = styled(Absolute)`
+  height: 400px;
+`
+
+storiesOf('Absolute', module)
+  .add('Over an image', () => (
+    <RelativeBox width={1 / 2}>
+      <Absolute top={8} left={0}>
+        <Flag>Hello Flag</Flag>
+      </Absolute>
```

Comment:
NB/FWIW: many times negative margins can achieve the same thing as positioning with fewer chances of CSS bugs